frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -528,7 +528,7 @@ export const NodesUpdatesGroup: React.FC<NodesUpdatesGroupProps> = ({
     <UpdatesGroup divided={divided}>
       <UpdatesType>
         <Link to={`/k8s/cluster/nodes?rowFilter-node-role=${machineConfigPool.metadata.name}`}>
-          {name} {NodeModel.labelPlural}
+          {t('public~{{name}} {{resource}}', { name, resource: NodeModel.labelPlural })}
```

Comment:
I would just use `Nodes` inline here so that the resource kind gets translated.

```suggestion
          {t('public~{{name}} Nodes', { name })}
```

Reply:
@spadgett done!